modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -1092,7 +1092,7 @@ definition {
 
 			LexiconEntry.gotoAdd();
 
-			Click(
+			ClickNoError(
```

Comment:
@thaiscabrals The failure happens when asserting `ObjectAdmin#VIEW_OBJECT_FIELD_LABEL`, so I am not sure if this will fix the test.